dev/core/src/com/google/gwt/core/ext/linker/impl/installScriptEarlyDownload.js

```diff
@@ -19,23 +19,15 @@ function installScript(filename) {
     var doc = getInstallLocationDoc();
     var docbody = doc.body;
     var script;
-    // for sourcemaps, we inject textNodes into the script element on Chrome
-    if (navigator.userAgent.indexOf("Chrome") > -1 && window.JSON) {
-      var scriptFrag = doc.createDocumentFragment()
-      // surround code with eval until crbug #90707 
-      scriptFrag.appendChild(doc.createTextNode("eval(\""));
+    // for sourcemaps, we inject the code as a single string for Chrome
+    if (navigator.userAgent.includes("Chrome")) {
```

Comment:
this [breaks ](https://caniuse.com/es6-string-includes) IE11 compatibility

Reply:
I am ashamed, reverted to indexOf